core/templates/pages/exploration-editor-page/services/exploration-tags.service.ts

```diff
@@ -24,13 +24,21 @@ angular.module('oppia').factory('ExplorationTagsService', [
   function(ExplorationPropertyService, TAG_REGEX) {
     var child = Object.create(ExplorationPropertyService);
     child.propertyName = 'tags';
+    /**
+      *@param {string} value - tag array to be normalized
```

Comment:
This should have a return value also?

Reply:
right! on it! sorry for the late response.